docs/platform-setup.md

```diff
@@ -18,3 +18,124 @@ For each job, disable the GitHub multibranch status:
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
+
+  - Complete all required fields and click on "Save"
+
+- Create a docker (hosted) nexus repository: `docker-registry`
+
+  - In Server Administration and Configuration > Repositories,  click on the "Create repository" button
+
+  - Complete all required fields and click on "Save"
+
+### Cleanup Policy Configuration
+
+For each nexus docker registry, create a new policy cleanup and  apply to it.
+
+#### Create New Cleanup Policy
+
+- In Server Administration and Configuration > Cleanup Policies,  click on the "Create Cleanup Policy" button
+
+- To keep all nuxeo builder and jenkins master images ( latest tag for example ) add this asset name matcher `~(.*nuxeo/builder-.*/manifests/latest|.*nuxeo/platform.*-jenkinsx/manifests/[0-9]{1,}.[0-9]{1,}.[0-9]{1,})`
+
+- Complete all required fields and click on "Save"
+
+#### Apply Cleanup Policy
+
+- Open an existing nexus repository configuration for instance [docker-registry](https://nexus.platform.dev.nuxeo.com/#admin/repository/repositories:docker-registry)
+
+- Go done to cleanup section and select the corresponding item
+
+- Click on "Save".
+
+### Create New Docker Ingress Service
+
+This ingress service will act as Reverse Proxy for nexus docker repository.
+The reverse proxy would direct docker commands ( pull, push or login ) received at
+
+`https://docker.platform.dev.nuxeo.com`
+
+to
+
+`https://nexus.platform.dev.nuxeo.com/repository/docker-registry/`
+
+Create an ingress configuration file following this [example](../templates/docker-ingress.json)
+
+Create new ingress service from the configuration file
+
+```bash
+kubectl create -f templates/docker-ingress.json
+```
+
+Check if the ingress service is ready
+
+```bash
+kubectl get -f templates/docker-ingress.json
+```
+
+List the catalog from the new docker registry under nexus
+
+```bash
+DOCKER_REGISTRY=$(kubectl get -f templates/docker-ingress.json -o json --output=jsonpath={.spec.rules[].host} )
+curl https://${DOCKER_REGISTRY}/v2/_catalog
+```
+
+### Update The Config.json Secret
+
+Create/update Docker auth for a given host(docker.platform.dev.nuxeo.com for example) and user in the config.json file
+
+```bash
+export NAMESPACE=platform
+NEXUS_PASSWORD=$(kubectl get secret nexus -n ${NAMESPACE} -o=jsonpath='{.data.password}' | base64 --decode)
+jx create docker auth --host "docker.${NAMESPACE}.dev.nuxeo.com" --user "admin" --secret "${NEXUS_PASSWORD}"
+```
+
+If building your images with Kaniko, you have to add the Kubernetes secret that contains the config.json Docker configuration (in all existing skaffold.yaml)
+
+```yaml
+cluster:
+    ...
+    dockerConfig:
+      secretName: jenkins-docker-cfg
+```
+
+### Update PodTemplates
+
+#### Create A Secret Based On Existing Docker Credentials
+
+We suppose you have a json file containing credentials for `docker.platform.dev.nuxeo.com`
```

Comment:
We should have it no? as it is the previous step.

Reply:
The previous if i'm not wrong  define the secret to be used by Kaniko.
In this section we are defining  the secret to be used by kubernetes to retrieve pod template image.

we will have two secret : 
 - jenkins-docker-cfg ( Opaque Secret used by Kaniko)
 - nexus-docker-cfg ( Generic Secret with kubernetes.io/dockerconfigjson type used by kubernetes to pull pod template image )

because they store the docker credentials with different  methods .
 - Kubernetes save the docker credential  on section '.dockerconfigjson'
 - Kaniko save the docker credential  on section 'config.json'

Reply:
Maybe it's worth adding a comment explaining this.

Reply:
Yes, definitely.

Reply:
changes applied